config/common.yaml

```diff
@@ -433,9 +433,7 @@ servers:
             service: "[[.KieName]]"
         spec:
           strategy:
-            rollingParams:
```

Comment:
What is the reason to change the strategy?
It seems unrelated to the JIRA reference in PR.

Reply:
this change needs more some tweaking, so ill remove it from this PR... i've noticed that envs relying on a persisted H2 DB, will have to use a Recreate strategy, otherwise rollouts never finish... these errors are thrown.
```shell
18:54:10,431 ERROR [org.jboss.jca.core.tx.jbossts.XAResourceRecoveryImpl] (Periodic Recovery) IJ000906: Error during crash recovery: java:/jboss/datasources/rhpam_EJBTimer (IJ031084: Unable to create connection): javax.resource.ResourceException: IJ031084: Unable to create connection
	at org.jboss.jca.adapters.jdbc.xa.XAManagedConnectionFactory.getXAManagedConnection(XAManagedConnectionFactory.java:531)
Caused by: org.h2.jdbc.JdbcSQLException: Database may be already in use: null. Possible solutions: close all other connection(s); use the server mode [90020-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
org.jboss.jca.adapters.jdbc.xa.XAManagedConnectionFactory.getXAManagedConnection(XAManagedConnectionFactory.java:515)
	... 16 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/opt/eap/standalone/data/rhpam.mv.db [1.4.193/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:766)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
```

Reply:
this only really affects rhpam authoring today, so ill change only that deployment to Recreate